src/app/projects/states/portfolio/directives/portfolio-welcome-step/portfolio-welcome-step.component.html

```diff
@@ -0,0 +1,29 @@
+<div class="card-wrapper">
```

Comment:
**Question**: is `card-wrapper` class from bootstrap?If it just not used can we remove it (I did not see any styling on scss file).